functions/Invoke-DbaDiagnosticQuery.ps1

```diff
@@ -217,18 +276,26 @@ Then it will export the results to Export-DbaDiagnosticQuery.
 
                 if (($QueryName.Count -ne 0) -and ($QueryName -notcontains $scriptpart.QueryName)) { continue }
                 if (!$scriptpart.DBSpecific -and !$DatabaseSpecific) {
+                    if ($ExportQueries) {
+                        New-Item -Path $OutputPath -ItemType Directory -Force *> $null
```

Comment:
$null = or [void] is more readable here

Reply:
Ok, if it's preferred to use `$null =` I will change that. I agree that's probably easier to read.